kubefwd/create-refresh-button.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
did you consider using the uibutton extension for this? (doesn't matter much for this PR, just wondering if the direction we want to go is every extension that makes a button does something like this or if this was just good enough for now)